client/blocks/comment-detail/index.jsx

```diff
@@ -17,6 +17,7 @@ import CommentDetailPost from './comment-detail-post';
 import CommentDetailReply from './comment-detail-reply';
 import { decodeEntities, stripHTML } from 'lib/formatting';
 import { getPostCommentsTree } from 'state/comments/selectors';
+import getSiteComment from 'state/selectors/get-site-comment';
```

Comment:
Oh curious, were there any bundle size benefits of this? Normally I see the following, which makes it easy to import multiple selectors.
```jsx
import { getSiteComment } from 'state/selectors';
```